src/View/BakeView.php

```diff
@@ -142,7 +138,10 @@ protected function _paths(?string $plugin = null, bool $cached = true): array
     {
         $paths = parent::_paths($plugin, false);
         foreach ($paths as &$path) {
-            $path .= static::BAKE_TEMPLATE_FOLDER . DS;
+            // Append 'bake' to all directories that aren't the application override directory.
+            if (strpos($path, 'plugin' . DS . 'Bake') === false) {
+                $path .= static::BAKE_TEMPLATE_FOLDER . DS;
```

Comment:
We couldn't agree on a way to extend View to support feature directories to replace this unfortunately. Something to do for cake 5.

Reply:
I think long-term we can actually just remove the bake/ directory in templates in the plugin and avoid the whole issue since templates/plugin/Bake will declutter everyone.